docs/ossec_alerts.md

```diff
@@ -40,7 +41,6 @@ These are the values you must specify in `prod-specific.yml`:
  * Email username to authenticate to the SMTP relay: `sasl_username`
  * Domain name of the email used to send OSSEC alerts: `sasl_domain`
  * Password of the email used to send OSSEC alerts: `sasl_password`
- * The fingerprint of your SMTP relay: `smtp_relay_fingerprint`
```

Comment:
When I said "3 supported tiers", I meant:

1. `secure` with default CA's
2. `secure` with custom CA
3. `fingerprint` verification

I agree that we should not bother supporting the `verify` option. At the moment, I do not see any compelling reason to do so.